en/development/testing.rst

```diff
@@ -208,6 +208,10 @@ expectations and assertions your tests make. Additionally because tests are
 code, they are easy to re-run whenever you make a change. This helps prevent
 the creation of new bugs.
 
+.. note::
+
+EventManager is refreshed for each test method. Meaning, that when running multiple tests at once, you will loose your event listeners that were registered in config/bootstrap.php as the bootstrap is only executed once.
```

Comment:
Yep, fixed. Sorry, haven't contributed to docs that much to able to remember guidelines :/